tests/roles/development_environment/tasks/main.yaml

```diff
@@ -46,3 +46,18 @@
 - name: debug token
   ansible.builtin.debug:
     var: before_adoption_token
+
+- name: creates credential and stores it
```

Comment:
would be nice to call it out in the logs more like 
"create credential for sanity checking its value after adoption"

or similar